sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_subscriptions_revised_changelog_v1/query.sql

```diff
@@ -97,6 +97,7 @@ WITH original_changelog AS (
         subscription.* REPLACE (
           STRUCT(
             JSON_VALUE(subscription.metadata.appliedPromotionCode) AS appliedPromotionCode,
+            JSON_VALUE(subscription.metadata.cancellation_reason) AS cancellation_reason,
```

Comment:
Does cancellation_reason need to be added to the metadata struct in the synthesize_subscription udf?  I don't see why it wouldn't be but the query is tough to follow

Reply:
Yes, it should be added there as well. Good catch!